<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationViewer</class>
 <widget class="QWidget" name="AnimationViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>527</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="LeftContainer">
     <item>
      <widget class="QGroupBox" name="gridCelDataBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Cel Data</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="customAttributes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Values</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label3">
             <property name="text">
              <string>Alpha</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="alphaSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label4">
             <property name="text">
              <string>Scale</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label5">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label6">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label7">
             <property name="text">
              <string>Position</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label8">
             <property name="text">
              <string>Rotation</string>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QComboBox" name="rotationTweenTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QComboBox" name="positionTweenTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QComboBox" name="scaleTweenTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QComboBox" name="alphaTweenTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="positionXSpinBox">
             <property name="minimum">
              <number>-9999</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="positionYSpinBox">
             <property name="minimum">
              <number>-9999</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="rotationXSpinBox">
             <property name="minimum">
              <number>-4096</number>
             </property>
             <property name="maximum">
              <number>4096</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="scaleXSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <double>-100.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QDoubleSpinBox" name="scaleYSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <double>-100.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="baseAttributes">
          <property name="title">
           <string>Base</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label7_3">
               <property name="text">
                <string>Priority</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="positionZSpinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>-9999</number>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label7_4">
               <property name="text">
                <string>Center</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="centerYSpinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>-9999</number>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label6_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="centerXSpinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>-9999</number>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label2">
               <property name="text">
                <string>BlendType</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="blendTypeComboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Alpha</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Add</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sub</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>HalfAdd</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label1">
               <property name="text">
                <string>Blur</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="blurSpinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Facing Option</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="facingOptionCombobox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Look at target</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Look at moving direction</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="relativeToTargetCheckBox">
             <property name="text">
              <string>Relative to Target</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox">
       <property name="title">
        <string>Animation Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="AnimationControls">
          <item>
           <widget class="QGroupBox" name="horizontalGroupBox_2">
            <property name="title">
             <string>SnapToGrid</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="snapGridCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxGridX">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxGridY">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="horizontalGroupBox_2">
            <property name="title">
             <string>Option</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>OnionSkin</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="animationControlBox">
          <item>
           <widget class="QToolButton" name="saveAnimationButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../ui.qrc">
              <normaloff>:/resource/disk.png</normaloff>:/resource/disk.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="showPaletteButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../ui.qrc">
              <normaloff>:/resource/palette.png</normaloff>:/resource/palette.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="playAnimationButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../ui.qrc">
              <normaloff>:/resource/play.png</normaloff>:/resource/play.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mainContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Animation Canvas</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="cellDataBox">
        <item>
         <layout class="QVBoxLayout" name="animationViewerContainer"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ui.qrc"/>
 </resources>
 <connections/>
</ui>
